test/run-tests

```diff
@@ -1279,6 +1267,25 @@ def main():
     # this is used in PR status - context name
     ansible_id = f"ansible-{ansible_version.version[0]}.{ansible_version.version[1]}"
 
+    # If test_collections is true, download lsr_role2collection.py
+    test_collections = config.get("test_collections", False)
+    if test_collections:
+        r2c_url = "https://raw.githubusercontent.com/linux-system-roles/auto-maintenance/master/lsr_role2collection.py"  # noqa:E501
+        r2c = requests.get(r2c_url, allow_redirects=True)
+        r2c_path = tempfile.mkdtemp()
+        role2collection_path = r2c_path + "/lsr_role2collection.py"
+        open(role2collection_path, "wb").write(r2c.content)
+        logging.debug(f"Write lsr_role2collection to {role2collection_path}")
+        sys.path.append(r2c_path)
+
+        os.environ["COLLECTION_SRC_OWNER"] = "linux-system-roles"
+        collection_dest_path = tempfile.TemporaryDirectory().name
+        os.environ["COLLECTION_DEST_PATH"] = collection_dest_path
+        # The tests dir is copied to the temporary dir
+        # which is outside of the collections.
+        os.environ["COLLECTION_TESTS_DEST_PATH"] = tempfile.TemporaryDirectory().name
+        logging.debug(f"Appended {r2c_path} to PYTHONPATH")
```

Comment:
The temp files and directories should be removed at the end

Reply:
Sorry, somehow I was thinking they were automatically removed at the end. They are not... >_<